### DISCUSSION TITLE
Text is grayed after I press 'f'. What I can do to fix it ?

### DISCUSSION THREAD

**Discussion by ilqvya** [LazyVim/LazyVim#4383] [General]
<img width="1066" alt="image" src="https://github.com/user-attachments/assets/5eee353a-57aa-436e-b1d5-59ffb77d1c91">

**Comment by dpetka2001** [comment]
It's the plugin `flash.nvim`. Read its [configuration docs](https://github.com/folke/flash.nvim#%EF%B8%8F-configuration) for how to customize it. You're looking for `modes.char` section. It replaces the default `f, F, t, T` Neovim motions.
